### PR TITLE
ADDED LOGIN

### PR DESCRIPTION
**Pull request by DSDmark**

### CODE REVIEW

File: server/controlles/authCtrl.ts [hunk 1]
```diff
@@ -68,6 +68,27 @@ const authCtrl = {
 			return res.status(500).json({ msg: error });
 		}
 	},
+	login: async (req: Request, res: Response) => {
```
Comment (DSDmark) [inline server/controlles/authCtrl.ts:71]: HERE IS ERROR OCCUES

File: server/controlles/authCtrl.ts [hunk 2]
```diff
@@ -68,6 +68,27 @@ const authCtrl = {
...
+		try {
+			const { account, password } = req.body;
+			const user = await User.findOne(account);
```
Comment (DSDmark) [inline server/controlles/authCtrl.ts:74]: PROMISE IS NOT RESOLVE

**Review by DSDmark — COMMENTED** [review]
THERE IS A LOGIN ERROR

**Review by DSDmarkSuc — APPROVED** [review]
co-auth

**Pull request merged**